weblate/vcs/git.py

```diff
@@ -660,18 +678,47 @@ def create_pull_request(self, origin_branch, fork_remote, fork_branch):
             head = fork_branch
         else:
             head = "{0}:{1}".format(fork_remote, fork_branch)
-        self.execute(
-            [
-                "pull-request",
-                "--force",
-                "--head",
-                head,
-                "--base",
-                origin_branch,
-                "--message",
-                self.get_merge_message(),
-            ]
+        pr_url = "{}/pulls".format(self.api_url())
+        r = requests.post(
+            pr_url,
+            headers={
+                "Accept": "application/vnd.github.v3+json",
+                "Authorization": "token {}".format(settings.GITHUB_TOKEN),
+            },
+            json={
+                "head": head,
+                "base": origin_branch,
+                "title": self.get_merge_message(),
```

Comment:
This code has just created it first pull requests on Hosted Weblate and this seems to be wrong - it injects whole message into the title, while it should be split into title and body/description, see https://github.com/WeblateOrg/weblate/pull/4427. I've fixed this in 389f1028691abdc0a16f3f9d934218a325d7baaa